Authentication/Login/View/LoginView.swift

```diff
@@ -27,80 +27,56 @@ internal final class LoginView: UIView, LoginViewType, NibLoadable {
     
     internal var emailTextField: UITextField { return emailTextFieldOutlet }
     @IBOutlet weak var emailTextFieldOutlet: UITextField! {
-        didSet {
-            emailTextFieldOutlet.placeholder = emailPlaceholderText
-        }
+        didSet { emailTextFieldOutlet.placeholder = emailPlaceholderText }
```

Comment:
The thing is the file was very long and swiftlint complained. I know it's not the best solution, but it was a little difference of lines.